netstandard/ref/netstandard.csproj

```diff
@@ -9,15 +9,17 @@
     <IsCoreAssembly>true</IsCoreAssembly>
     <!-- disable obsolete warnings/errors -->
     <NoWarn>$(NoWarn);0618;0619;0809</NoWarn>
-
     <DefaultBuildAllTarget>Build</DefaultBuildAllTarget>
     <BuildAllProjects>true</BuildAllProjects>
+    <SignType>oss</SignType>
```

Comment:
Why do we need the SignType set explicitly here? We also shouldn't force it to be OSS it should default to real from the targets https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/sign.targets#L29. As it is a fully signed binary.